aspnetcore/web-api/http-repl.md

```diff
@@ -804,6 +804,88 @@ To set an HTTP request header, use one of the following approaches:
   https://localhost:5001/people~ set header Content-Type
   ```
 
+## Test secured endpoints
+
+The HTTP REPL supports the testing of secured endpoints through the use of HTTP request headers. Examples of supported authentication and authorization schemes include basic authentication, JWT bearer tokens, and digest authentication. For example, send a bearer token to an endpoint with the following command:
+
+```console
+set header Authorization "bearer <TOKEN VALUE>"
+```
+
+To access an Azure-hosted endpoint or to use the [Azure REST API](/rest/api/azure/), you need a bearer token. Use the following steps to obtain a bearer token for your Azure subscription and to retrieve a list of your Azure App Service instances:
```

Comment:
Would it help to make it slightly more clear in this paragraph which parts are done by HttpRepl and which are just set up? Something like (wordsmithing probably needed):
> Use the following steps to obtain a bearer token for your Azure subscription via the Azure CLI, and then use that bearer token in the HTTP REPL to retrieve a list of your Azure App Service instances?